frontend/src/components/CodingChallengeCard.tsx

```
@@ -1,32 +1,70 @@
-            role="button"
-        >
-            <h4>{title}</h4>
-            <p>{unit}</p>
-        </div>
+        <>
+            <div>
+                <div
+                    className={styles.challengeCard}
+                    onClick={handleClick}
+                    role="button"
+                >
```

Biome (1.9.4)

[error] The HTML element with the interactive role "button" is not focusable.

A non-interactive HTML element that is not focusable may not be reachable for users that rely on keyboard navigation, even with an added role like "button".
Add a tabIndex attribute to make this element focusable.

(lint/a11y/useFocusableInteractive)